service/src/main/java/bio/terra/pipelines/stairway/imputation/AddWdsRowStep.java

```diff
@@ -31,6 +33,8 @@ public AddWdsRowStep(WdsService wdsService, SamService samService) {
   }
 
   @Override
+  @SuppressWarnings("java:S2259") // suppress warning for possible NPE - we do validate not null in
```

Comment:
hmm what was it complaining when using `validateRequiredEntries` that it wasnt complaining about before?

Reply:
it's saying `pipelineName` might be null in row 78. but `FlightUtils.validateRequiredEntries` makes sure that each checked entry is not null. of course the checker doesn't know that, though.